#! /bin/sh /usr/share/dpatch/dpatch-run
## missing_headers.dpatch by Samuel Mimram <smimram@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Include missing header files.

@DPATCH@
diff -urNad linphone-1.3.5~/console/commands.c linphone-1.3.5/console/commands.c
--- linphone-1.3.5~/console/commands.c	2006-03-04 11:17:10.000000000 +0000
+++ linphone-1.3.5/console/commands.c	2006-05-15 11:04:47.000000000 +0000
@@ -30,6 +30,7 @@
 #include <limits.h>
 #include <ctype.h>
 #include <unistd.h>
+#include <readline/readline.h>
 #include <linphonecore.h>
 #include "linphonec.h"
 
diff -urNad linphone-1.3.5~/console/linphonec.c linphone-1.3.5/console/linphonec.c
--- linphone-1.3.5~/console/linphonec.c	2006-03-16 17:17:40.000000000 +0000
+++ linphone-1.3.5/console/linphonec.c	2006-05-15 11:04:30.000000000 +0000
@@ -31,6 +31,8 @@
 #include <string.h>
 #include <signal.h>
 #include <limits.h>
+#include <readline/readline.h>
+#include <readline/history.h>
 #include <linphonecore.h>
 #include "linphonec.h"
 #include "../config.h"
diff -urNad linphone-1.3.5~/exosip/eXutils.c linphone-1.3.5/exosip/eXutils.c
--- linphone-1.3.5~/exosip/eXutils.c	2005-09-09 10:47:05.000000000 +0000
+++ linphone-1.3.5/exosip/eXutils.c	2006-05-15 11:04:11.000000000 +0000
@@ -218,6 +218,7 @@
 #include <sys/param.h>
 
 #include <stdio.h>
+#include <errno.h>
 
 static int ppl_dns_default_gateway_ipv4 (char *address, int size);
 static int ppl_dns_default_gateway_ipv6 (char *address, int size);
diff -urNad linphone-1.3.5~/exosip/jcallback.c linphone-1.3.5/exosip/jcallback.c
--- linphone-1.3.5~/exosip/jcallback.c	2006-03-08 11:48:57.000000000 +0000
+++ linphone-1.3.5/exosip/jcallback.c	2006-05-15 11:04:11.000000000 +0000
@@ -41,6 +41,8 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <netdb.h>
+
+#include <errno.h>
 #endif
 
 #include <eXosip.h>
diff -urNad linphone-1.3.5~/exosip/udp.c linphone-1.3.5/exosip/udp.c
--- linphone-1.3.5~/exosip/udp.c	2006-02-18 14:40:19.000000000 +0000
+++ linphone-1.3.5/exosip/udp.c	2006-05-15 11:04:11.000000000 +0000
@@ -36,9 +36,10 @@
 #endif
 #else
 #include <windows.h>
-#include <errno.h>
 #endif
 
+#include <errno.h>
+
 extern eXosip_t eXosip;
 
 /* Private functions */
